prep-html-skeleton/index.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html></html>
```

Comment:
Hi Daniel, be sure to put the closing tag for the `<html>` element at the end of the file so that everything (except DOCTYPE) is inside of it.